cypress/integration/lesson14_spec.js

```diff
@@ -0,0 +1,34 @@
+describe('My First Test', () => {
+  before(() => {
+    cy.visit('https://demoqa.com')
+    })
+
+    after(() => {
+      cy.reload(); // cleanup text from the inputs fields
+    });
+      // Should be on a new URL which includes '/elements'
```

Comment:
Delete this comments. U have test name, it's enough
Your test name should be more meaningfull

Reply:
Accepted.